tests/01_unit/scenes/atmosphere/test_molecular_atmosphere.py

```diff
@@ -137,3 +138,37 @@ def test_molecular_atmosphere_switches(
             has_scattering=False,
             error_handler_config=absorption_database_error_handler_config,
         )
+
+
+def test_molecular_atmosphere_depolarization(mode_ckd):
+    atmosphere = MolecularAtmosphere(rayleigh_depolarization=0.5)
+    si = default_spectral_index(atmosphere)
+    depol = atmosphere.eval_depolarization_factor(si)
+    template, _ = traverse(atmosphere)
+    assert template.render(KernelContext(si=si))
+    assert isinstance(depol, pint.Quantity)
+    assert len(depol) == 1
```

Comment:
How about turning this into a shape check? (you can use `np.shape()` if `depol` is not an array).